_docs/features/finance.md

```diff
@@ -14,3 +14,13 @@ You may see financial data appear in your search results if it’s particularly
 | NYSE          | NYSE                          | 15 minutes*             |
 
 \* Delay applies during trading hours only. Outside of trading hours, financial data reflects close of previous trading session.
+
+--------------------------------------
+
+**S&P Dow Jones Indices Terms and Disclaimers**
```

Comment:
Rather than a rule and bolded text, can we use an `<h3 />`? That'd be more semantically correct.

```
### S&P Dow Jones Indices Terms and Disclaimers
```

Reply:
Ah! That's a great idea, thanks!
Amended the previous commit with your suggested changes :+1: